src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
```

Comment:
Prop user is `User`, not `User | null` nor `User?`
```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
```